common/travis/build.sh

```diff
@@ -6,26 +6,12 @@ if [ "$1" != "$2" ]; then
 	arch="-a $2"
 fi
 
-# Tell xbps-src what is our arch, this is done when doing
-# binary-bootstrap, but we need to do it every time since
-# our masterdir is ethereal.
-# /bin/echo -e '\x1b[32mWriting bootstrap arch into .xbps_chroot_init of masterdir\x1b[0m'
-# echo "$1" > /hostrepo/masterdir/.xbps_chroot_init
-
-/bin/echo -e '\x1b[32mPreparing chroot with chroot_prepare()\x1b[0m'
-source hostrepo/common/xbps-src/shutils/chroot.sh || {
-	echo "Failed to source chroot.sh for chroot_prepare()" >&2 ;
-	exit 1
-}
-
-XBPS_SRCPKGDIR=/hostrepo/srcpkgs XBPS_MASTERDIR=/ chroot_prepare $1 || {
-	echo "Failed to prepare chroot!" >&2 ;
-	exit 1
-}
-
-# Two times due to updating xbps itself
-/hostrepo/xbps-src -H "$HOME"/hostdir bootstrap-update
-/hostrepo/xbps-src -H "$HOME"/hostdir bootstrap-update
+# Make sure `base-chroot` is really up-to-date
+/hostrepo/xbps-src -H "$HOME"/hostdir -f pkg base-chroot || exit 1
```

Comment:
Can you add a comment here as to why `-f` is necessary?

Reply:
Probably not necessary but does not harm, just to make sure it's built.

Reply:
probably not necessary, but does not make any harm.